bindings/rust/aws-lc-fips-sys-template/build_bindgen.rs

```diff
@@ -0,0 +1,103 @@
+// SPDX-License-Identifier: Apache-2.0 OR ISC
```

Comment:
Minor comment, not blocking on the PR. But we should migrate this to a build folder similar to [openssl-sys](https://github.com/sfackler/rust-openssl/tree/master/openssl-sys/build). That way we don't have have multiple build related files at the root of the crate. You can specify this separate location in the [`Cargo.toml`](https://github.com/sfackler/rust-openssl/blob/master/openssl-sys/Cargo.toml#L14). Just keeps the crate root cleaner.

Reply:
That's a good point. I'm planning to add `README.md`s for both sys crates in the next PR. I'll apply the recommended directory shuffling along with that change then.